src/metabase/public_settings.clj

```diff
@@ -224,8 +224,7 @@
 (defsetting enable-query-caching
   (deferred-tru "Enabling caching will save the results of queries that take a long time to run.")
   :type    :boolean
-  :default false
-  :visibility :settings-manager)
+  :default false)
```

Comment:
@noahmoss I'm reverting your recent change to the setting because it caused issues.